.gitignore

```diff
@@ -1,2 +1,5 @@
 /projects/unix/_obj*/
 /projects/unix/mupen64plus-rsp-hle*.so
+/projects/unix/mupen64plus-rsp-hle*.dll
```

Comment:
I’m not sure to understand this line ? How a `.dll` can reach the `unix` folder?

Reply:
Building in msys2/mingw can cause a dll to end up there, I use the Makefiles for RMG on windows too.

Reply:
See https://github.com/mupen64plus/mupen64plus-rsp-hle/blob/47f8e92e9a8bb9e49195ecfb03e9df0b63add242/projects/unix/Makefile#L62-L64